NineChronicles.Headless/GraphTypes/ActionQuery.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace NineChronicles.Headless.GraphTypes
 {
-    public class ActionQuery : ObjectGraphType
+    public partial class ActionQuery : ObjectGraphType
```

Comment:
Frankly, I'm not a big fan of `partial` class since usually it can be a signal of abstraction lacking. (i.e., if we need to `partial` to maintain some class, It can also mean that the class has too many features)

it might be better to consider subgraphs, even if on the next PR. 🤔

Reply:
Just as you said, there are too many features to put altogether into the one class.
But I wonder subgraph is the answer cuz all actionQueries are on the same hierarchy level I think.
I'll make an issue based on this command and we can continue discussion